Shorten list of other matching cards if necessary

hgarus opened the pull request:
Normally I would prefer splitting the message text, but discord.js split option causes the file attachment to be shown multiple times. I think the case of the card list being really long is rare enough that we can just not show the whole list. if it is too long.

The real fix for this issue would probably consist of a search which doesn't list "Steelshaper Apprent**ice**" and "Phyrexian Spl**ice**r as results when searching for ice.

hcschild committed:
Revert "Merge branch 'develop' of https://github.com/bra1n/judgebot into develop"

This reverts commit 3b0199935aedfc4ca4714ae5817a045d636e0264.

hcschild committed:
Merge branch 'develop' of https://github.com/bra1n/judgebot into develop

hgarus committed:
Shorten list of other matching cards if necessary

Fixes #26

Don't display all matching cards to keep the message length below discord's character
limit of 2000.

bra1n commented on modules/card.js:
```diff
@@ -6,6 +6,7 @@ class MtgCardLoader {
     constructor() {
         this.cardApi = "https://api.magicthegathering.io/v1/cards?name=";
         this.commands = ["card"];
+        this.maxLength = 2000;
```
Why not just limit the number of suggested cards to something like 5-10? Nobody wants to see more than 10 alternative cards anyway. ;)

hgarus replied:
Question is how we determine those 10. I want to tackle better search at some point, but I can't promise when I get around to doing that.

bra1n replied:
Just use the first 10 for now?

hcschild committed:
Merge branch 'develop' of https://github.com/bra1n/judgebot into develop

# Conflicts:
#	aggregators/mtg_cards/cards.js

hcschild committed:
Implemented Feature #28
!card now shows if the card is legal in legacy, vintage, modern, standard or commander. When the format is not listed the card wasn't printed in a set legal for this format.

bra1n committed:
Update README.md

hcschild committed:
changed output for feature #28
reworked ipg parser for #11

hcschild committed:
changed output for feature #28
reworked ipg parser for #11
removed unused variable maxLength

bra1n committed:
added codeclimate configs

hcschild committed:
Added list of available topics and added a constant to set the maximum topics.
Removed redundant return.

hcschild committed:
Added list of available topics and added a constant to set the maximum topics.
Removed redundant return.

bra1n committed:
fixed IPG handling

bra1n committed:
Merge pull request #29 from bra1n/ipg_shortener

reworked ipg parser for #11
